MzML/MzmlDynamicData.cs

```diff
@@ -147,7 +147,7 @@ public override MsDataScan GetOneBasedScanFromDynamicConnection(int oneBasedScan
                                     case "MS:1000128":
                                         isCentroid = false;
                                         throw new MzLibException("Reading profile mode mzmls not supported");
-                                        break;
+                                        //break;
```

Comment:
We never reach the break because of a throw. VS gives an annoying warning about it, so I commented it out. I could have deleted it, but wanted to keep the break incase we ever remove the exception.